scripts/lib/libvirt/compute-config

```diff
@@ -32,6 +32,8 @@ def main():
                         help="Virtual Disk Directory (e.g /dev/cloud)")
     parser.add_argument("bootorder", type=str,
                         help="Boot Order (e.g. 2)")
+    parser.add_argument("numcontrollers", type=str,
+                        help="Number of controller nodes")
```

Comment:
@MaximilianMeister I do not get your point: I call all my Python scripts from the shell in bash.  The thing here is the meaning of those different parameters and the operations that we do with them. In this case are are expected integers (see the `>` bellow that relate both parameters)

Also argparse takes care to converting this for us, even if in `sys.argv` everything is a string.